src/character.h

```diff
@@ -484,6 +484,7 @@ class Character : public Creature, public visitable<Character>
         std::vector<bionic> my_bionics;
 
     protected:
+        virtual void on_stat_change( const std::string &, int ) override {};
```

Comment:
Sure. I'm not arguing, but I have an example which made such statements possible. It has nothing to do with JSON: `void Creature::mod_stat( const std::string &stat, int modifier )` and `void Character::mod_stat( const std::string &stat, int modifier )`. Probably It has to be replaced too.